# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: accessible/basetypes/Accessible.h
# Commit: c6db418af190
# Full Hash: c6db418af19097f562d94b87bbf25362556b2fbc
# Author: Morgan Rae Reschenberg <mreschenberg@mozilla.com>
# Date: 2022-10-21 08:39:18
# Regressor Bug: 1787284
# File Overlap Count: 1
# Description:
#   Bug 1787284: [Part 2] Cache name and create radio name pivot rule r=eeejay
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D159118
# ==============================================================================

diff -r fda6706bb580 -r c6db418af190 accessible/basetypes/Accessible.h
--- a/accessible/basetypes/Accessible.h	Thu Oct 20 21:40:05 2022 +0000
+++ b/accessible/basetypes/Accessible.h	Thu Oct 20 21:40:06 2022 +0000
@@ -501,6 +501,8 @@
 
   bool IsHTMLOptGroup() const { return mType == eHTMLOptGroupType; }
 
+  bool IsHTMLRadioButton() const { return mType == eHTMLRadioButtonType; }
+
   bool IsHTMLTable() const { return mType == eHTMLTableType; }
   bool IsHTMLTableRow() const { return mType == eHTMLTableRowType; }
 